readmes/README.Linux.md

```diff
@@ -1,108 +1,173 @@
-## For CentOS:
+# Set up development environment for Cloudberry Database on Linux
 
-- Install Dependencies
+This document shares how to set up the development environments on different Linux systems (CentOS 7, RHEL, and Ubuntu). Before you start to [explore the Cloudberry Database](https://hackmd.io/XIgu6X8TTLWFmCttr2LQ6Q?view), follow this document to download and install the required tools and dependencies, and create demo user.
```

Comment:
> [explore the Cloudberry Database](https://hackmd.io/XIgu6X8TTLWFmCttr2LQ6Q?view)
The link is wrong, please update it.

Reply:
Thank you for your correction. May I ask what changes should be made to this link?

Reply:
Hi @TomShawn, any ideas?

Reply:
With @TomShawn assistance, I have corrected the link.